notification.py

```diff
@@ -622,8 +626,55 @@ def notify(self, pokemon, time_of_day):
                     n=name, s=mean))
                 return False
 
-        notified = Notification(pokemon_id, coordinates, time_till_hidden, iv, moves, iv_score, time_of_day).notify()
-        if notified:
+        whpushed = False
+        if config.WEBHOOKS:
+            whpushed = self.webhook(pokemon, time_till_hidden)
+
+        notified = False
+        if config.PB_API_KEY or config.TWITTER_ACCESS_KEY:
+            notified = Notification(pokemon_id, coordinates, time_till_hidden, iv, moves, iv_score, time_of_day).notify()
+
+        if notified or whpushed:
             self.last_notification = monotonic()
             self.recent_notifications.append(encounter_id)
-        return notified
+        return notified or whpushed
+
+    def webhook(self, pokemon, time_till_hidden):
+        """ Send a notification via webhook
+        """
+        if isinstance(time_till_hidden, (tuple, list)):
+            time_till_hidden, _ = time_till_hidden
```

Comment:
```python
time_till_hidden = time_till_hidden[0]
```
would be a little more clear I think, and a bit less fragile. Your assignment would break if the tuple/list had anything other than 2 items (which shouldn't happen, but we might as well guard against it since I think the code is clearer anyways).